### PR TITLE
feat:added badge component

### PR DESCRIPTION
**Pull request by KanekarSnehal**
Added badge component of 2 types 
1-profile badge
2-number badge

badge

### CODE REVIEW

**Comment by shivanigangadharan** [comment]
Badge added in both icons and avatar, looks good. 👍

**Comment by parthgabani777** [comment]
code looks clean.

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.badge-container {
+    display: flex;
+    flex-direction: row;
```
Comment (MaruthiPothuganti) [inline components/badge.css:3]: suggestion: flex-flow: row wrap;  can be used

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended: No major changes, Suggested Short hand property

File: components/badge.css (hunk 1 above)
Comment (KanekarSnehal) [reply components/badge.css:3]: Hey thanks for your suggestion, but I like it to be more descriptive that's why didn't used shorthand.

**Pull request merged**